### PR TITLE
Docs: Improved sm64ex advanced setup docs

### PR DESCRIPTION
**Pull request by kethinov**
This edit clarifies some things that are not obvious in the version that is currently live on the site.

This should prevent others from needing to go spelunking in Discord chat history to figure out how to do advanced builds.

### CODE REVIEW

File: worlds/sm64ex/docs/setup_en.md [hunk 1]
```diff
@@ -29,13 +29,22 @@ Then continue to `Using the Launcher`
...
+1. Go to the page linked for SM64AP-Launcher, and press on the topmost entry.
 2. Scroll down, and download the zip file for your OS.
-3. Unpack the zip file in an empty folder
+3. Unpack the zip file in an empty folder.
 4. Run the Launcher. On first start, press `Check Requirements`, which will guide you through the rest of the needed steps.
     - Windows: If you did not use the default install directory for MSYS, close this window, check `Show advanced options` and reopen using `Re-check Requirements`. You can then set the path manually.
-5. When finished, use `Compile default SM64AP build` to continue
-    - Advanced user can use `Show advanced options` to build with custom makeflags (`BETTERCAMERA`, `NODRAWINGDISTANCE`, ...), different repos and branches, and game patches such as 60FPS, Enhanced Moveset and others.
+5. When finished, use `Compile default SM64AP build` to continue.
+    - **Advanced configuration:** If you want to use off-by-default game options such as Better Camera, No Drawing Distance, etc or apply game patches such as 60FPS, Enhanced Moveset, etc, then use the `Compile custom build` option:
```
Comment (nicholassaylor) [inline worlds/sm64ex/docs/setup_en.md:38]:
```suggestion
    - **Advanced configuration:** If you want to use additional build options such as Better Camera, No Drawing Distance, etc or apply game patches such as 60FPS, Enhanced Moveset, etc, then use the `Compile custom build` option:
```

**Review by nicholassaylor — APPROVED** [review]
Having guided many people throuh this process, I can confirm that the information here is correct. One minor working nitpick, otherwise good!

**Comment by kethinov** [comment]
Thanks! 😀

File: worlds/sm64ex/docs/setup_en.md [hunk 2]
```diff
@@ -29,13 +29,22 @@ Then continue to `Using the Launcher`
...
-    - Advanced user can use `Show advanced options` to build with custom makeflags (`BETTERCAMERA`, `NODRAWINGDISTANCE`, ...), different repos and branches, and game patches such as 60FPS, Enhanced Moveset and others.
+5. When finished, use `Compile default SM64AP build` to continue.
+    - **Advanced configuration:** If you want to use additional build options such as Better Camera, No Drawing Distance, etc or apply game patches such as 60FPS, Enhanced Moveset, etc, then use the `Compile custom build` option:
+      - Set a name for your build, e.g. "archipelago" or whatever you like.
+      - Press the `Download Files` button.
+      - Set Make Flags, e.g. `-j8 BETTERCAMERA=1 NODRAWINGDISTANCE=1` to enable Better Camera and No Drawing Distance.
+      - Press `Apply Patches` to select patches to apply. Example patches include:
+        - 60FPS: Improves frame rate.
+        - Enhanced Moveset: Gives Mario new abilities. [Details here](https://github.com/TheGag96/sm64-port).
+        - Nonstop Mode Always Enabled: Makes it possible to fetch multiple stars in a level without exiting the level first.
```
Comment (N00byKing) [inline worlds/sm64ex/docs/setup_en.md:45]: Nit: Just "Nonstop Mode"

**Review by N00byKing — APPROVED** [review]

**Comment by nicholassaylor** [comment]
Just so you're aware, this PR has conflicts now that need to be resolved

**Comment by kethinov** [comment]
Thanks, fixed.

**Review by NewSoupVi — APPROVED** [review]

**Pull request merged**